### PR TITLE
Feature/presenter

### PR DESCRIPTION
**Pull request by eliasreis54**

### CODE REVIEW

File: lib/presentation/pages/presenter.dart [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  static TimeOfDay endTime;
+  static bool repeat;
+
+  static void selectListWeekDay(Weekday day, bool selected) {
+    selected
+        ? listWeekDay.add(day)
+        : listWeekDay.remove(day);
+  }
+
+  static void saveTask() {
```
Comment (mugbug) [inline lib/presentation/pages/presenter.dart:18]: Why static methods and properties?

File: lib/presentation/pages/presenter.dart [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    Task t = Task(
+        name: taskName,
+        status: TaskStatus.pending,
+        recurrence: listWeekDay,
+        endDate: DateTime.now(),
+        startDate: DateTime.now(),
```
Comment (mugbug) [inline lib/presentation/pages/presenter.dart:24]: Shouldn't we use the values selected by the user?

Comment (eliasreis54) [reply lib/presentation/pages/presenter.dart:24]: Yep, but the select dialog is not a DateTime is a TimeOfDay... I'll chage the model to be TimeOfDay

File: lib/presentation/pages/presenter.dart (hunk 1 above)
Comment (eliasreis54) [reply lib/presentation/pages/presenter.dart:18]: We does not need an instance of this class, i thought this class will only store the data and call the database service.

**Review by mugbug — APPROVED** [review]

**Pull request merged**